defaults/main.yml

```diff
@@ -1,12 +1,12 @@
-crayfish_user: www-data
+crayfish_user: "{% if ansible_os_family == 'RedHat' %}apache{% else %}www-data{% endif %}"
```

Comment:
If we can, I'd like to have the variables work like we were talking about in https://github.com/Islandora-Devops/ansible-role-karaf/pull/1

Reply:
I can do that once the Karaf PR is put to bed.

Reply:
@seth-shaw-unlv Both https://github.com/Islandora-Devops/ansible-role-karaf/pull/1 and https://github.com/Islandora-Devops/ansible-role-karaf/pull/2 are in.